modules/SpyUtils.js

```diff
@@ -4,6 +4,10 @@ import { isFunction } from './TestUtils'
 
 const noop = () => {}
 
+const engineSupportsConfigurableFunctionLength = (function () {
+  return Object.getOwnPropertyDescriptor(() => {}, 'length').configurable
```

Comment:
I guess I should do this? https://github.com/es-shims/es5-shim/blob/master/es5-shim.js#L77